core/qbft/qbft.go

```diff
@@ -87,18 +99,21 @@ type Msg struct {
 type uponRule int64
 
 const (
-	uponUnknown uponRule = iota
-	uponValidPrePrepare
-	uponQuorumPrepare
-	uponQuorumCommit
-	uponMinRoundChange
-	uponQuorumRoundChange
+	uponNothing uponRule = iota
+	uponJustifiedPrePrepare
+	uponUnjustPrePrepare
+	uponQuorumPrepares
+	uponQuorumCommits
+	uponUnjustRoundChange
+	uponFPlus1RoundChanges
+	uponQuorumRoundChanges
 )
 
-// Run returns the consensus decided value (Qcommit) or a context closed error.
-func Run(ctx context.Context, d Definition, t Transport, instance, process int64, inputValue []byte) (value []byte, err error) {
+// Run executes the consensus algorithm until the context closed.
+//nolint:gocognit // If is indeed a complex algorithm.
+func Run(ctx context.Context, d Definition, t Transport, instance []byte, process int64, inputValue []byte) (err error) {
```

Comment:
Suggest: parameter names can be `def Defination, trn Transport`

Reply:
sure, but `d` and `t` also works no?

Reply:
def sounds more of defination and t confuses with t of testing